client/src/components/Upload.vue

```diff
@@ -74,11 +74,16 @@ export default {
         result: null
       }));
       this.pendingUploads.push({
-        name: defaultFilename.replace(/\..*/, ""),
+        name:
+          files.length > 1
+            ? defaultFilename.replace(/\..*/, "")
+            : defaultFilename,
         files: files,
         type,
         fps: this.defaultFPS,
-        uploading: false
+        pipeline: null,
```

Comment:
Why add this?

Reply:
I think it was before I removed the run pipeline stuff.  My initially version had the pipeline stuff still in there.  Good catch, I'll remove it.